utils/node_local_test.py

```diff
@@ -4994,14 +4998,22 @@ def _explain():
             show_memleaks = False
             fi_signal = -rc
 
+        if self._aft.ignore_busy and self._aft.check_daos_stderr:
+            stderr = self._stderr.decode('utf-8').rstrip()
+            for line in stderr.splitlines():
+                if line.endswith(': Device or resource busy (-1012)'):
```

Comment:
nit: you probably only need to check for the bracketed error code here rather than a text representation as well

Reply:
We expect output to be well-formed so if the test fails because something prints -1012 but not the full text then that's probably something we want to fail.